setuptools/dist.py

```diff
@@ -636,6 +637,17 @@ def dash_to_underscore_warning(self, opt, section):
                 % (opt, underscore_opt))
         return underscore_opt
 
+    def uppercase_warning(self, opt, section):
```

Comment:
This method name is too broad. It's usually better to (1) be more specific about what it actually does and (2) use a verb because methods/functions do some actions, unlike variables/properties that only hold some values:
```suggestion
    def make_option_lowercase(self, opt, section):
```